test/spec/features/vpn/remove_spec.rb

```diff
@@ -6,18 +6,6 @@
     wait_until_container_gone('vpn.server-1')
   end
 
-  after(:each) do
-    run 'kontena stack rm --force vpn'
-  end
-
-  it 'removes the vpn stack' do
-    run 'kontena vpn create'
-    k = run 'kontena stack rm --force vpn'
-    expect(k.code).to eq(0)
-    k = run 'kontena stack show vpn'
-    expect(k.code).not_to eq(0)
```

Comment:
A spec for "`kontena stack show ...` fails after `kontena stack rm --force ...`" doesn't need to use the VPN stack.